worker-build/src/main.rs

```diff
@@ -104,10 +104,7 @@ fn copy_generated_code_to_worker_dir() -> Result<()> {
 }
 
 fn write_worker_shims_to_worker_dir() -> Result<()> {
-    // write some shims to make our wasm worker-compatible
-    // this is handled by higher-level crates automatically
```

Comment:
Why remove this comment?

Reply:
it could probably be expanded upon rather than removed.. @caass care to add some detail? 

primarily, the first comment is about the same as the fn name, but definitely could go further into detail about what compatibility means, and what the shim needs to do in order to bridge the js/wasm code. 

the second comment might be more helpful if it's expanded to contain the names of the crates, or what "this" is. are we talking about the Rust -> wasm compilation, or the codegen from wasm_bindgen to do memory stuff, or js module stuff?

Reply:
going to merge as-is and if we want to clarify later we can open a new PR.